ote/src/cljs/ote/style/base.cljs

```diff
@@ -51,9 +43,21 @@
                           ::stylefy/mode {:hover {:border-bottom (str "1px solid " colors/primary)
                                                   :margin-bottom "-1px"}}})
 
+(def base-button (merge
+                   {:padding-left "1.1em"
+                         :padding-right "1.1em"
+                         :text-transform "uppercase"
+                         :color "#FFFFFF"
+                         :background-color "#1565C0"
+                         :font-size "12px"
+                         :font-weight "bold"}
+                   blue-link-with-icon))
```

Comment:
This feels unnecessary. "base button" style will now allways have border button on hover?

Reply:
Some WIP code which should be removed, will do.